sound/soc/sof/xtensa/core.c

```diff
@@ -109,36 +109,19 @@ static void xtensa_stack(struct snd_sof_dev *sdev, void *oops, u32 *stack,
 {
 	struct sof_ipc_dsp_oops_xtensa *xoops = oops;
 	u32 stack_ptr = xoops->stack;
+	/* 4 dwords + 3 spaces + 1 terminating NUL */
+	unsigned char buf[8 * 4 + 3 + 1];
```

Comment:
one doubt here: is this really 8 bytes per value? You seem to use 16 + 3 + 1 with the format you selected:
0x0049fbb0: 8000f2d0 0049fc00 6f6c6c61 00632e63

Something's not adding up, not sure what I am missing or if this is a mistake

Reply:
0x0049fbb0: 8000f2d0 0049fc00 6f6c6c61 00632e63, here "0x0049fbb0: " is not generated by hex_dump_to_buffer, it is by "dev_err(sdev->dev, "0x%8.8x:".   Only "8000f2d0 0049fc00 6f6c6c61 00632e63" is generated by hex_dump_to_buffer.

Reply:
hex_dump_to_buffer does't create prefix of address.